pages/1.11/installing/upgrading/index.md

```diff
@@ -168,6 +172,28 @@ To update a cluster from permissive security to strict security, complete the fo
 
 1.  Go to the DC/OS Master [procedure](#masters) to complete your installation.
 
+## Bootstrap Node for Open Source users [open source type="inline" size="small" /]
```

Comment:
Tag misformatted. Use title-style tag instead.

Reply:
Hey @anithamuthu79 is this the line you were asking about when you mentioned the title tag? If so, I mean the bracketed style tag that surrounds the title, rather than the inline style tag that is there now. I'm not finding an example but i think it looks like:
```
[enterprise]
Your Title Here
[/enterprise] 
```

Reply:
@judithpatudith [OSS: true] is the Open Source tag used in the beginning of OSS page. Example:
/pages/1.11/installing/oss/cloud/index.md

I'll experiment using the below formats and see which looks appropriate.
[open source]
Your Title Here
[/open source] 

OR 

[OSS: true]